File: app-container-example-go/containers/AppExampleContainer.go
package containers

import (
	"github.com/exproletariy/pip-services3-containers-examples/app-container-example-go/build"
	cproc "github.com/pip-services3-go/pip-services3-container-go/container"
	rpcbuild "github.com/pip-services3-go/pip-services3-rpc-go/build"
)

type AppExampleContainer struct {
	cproc.Container
}

func NewAppExampleContainer() *AppExampleContainer {
	c := AppExampleContainer{}
	c.Container = *cproc.NewContainer("app-example", "Example of simple container")
	c.AddFactory(build.NewAppExampleServiceFactory())
	c.AddFactory(rpcbuild.NewDefaultRpcFactory())
	return &c
}

File: app-container-example-nodex/src/container/AppExampleContainer.ts
import { Container } from 'pip-services3-container-nodex';
import { DefaultRpcFactory } from 'pip-services3-rpc-nodex';
import { AppExampleServiceFactory } from "../build";

export class AppExampleContainer extends Container {
    public constructor() {
        super('app-example', 'Example of simple container');
        this._factories.add(new AppExampleServiceFactory());
        this._factories.add(new DefaultRpcFactory());
    }
}